pom.xml

```diff
@@ -0,0 +1,579 @@
+        <jsr305.version>3.0.2_1</jsr305.version>
+        <guava.version>23.0</guava.version>
+        <cxf.version>3.3.1</cxf.version>
+        <immutables.version>2.7.1</immutables.version>
+        <jackson.version>2.9.8</jackson.version>
+        <micrometer.version>1.0.6</micrometer.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <spring.boot.version>2.1.4.RELEASE</spring.boot.version>
+        <spring.version>5.1.6.RELEASE</spring.version>
+        <hibernate.validator.version>6.0.16.Final</hibernate.validator.version>
+        <annotation.api.version>1.3.2</annotation.api.version>
+        <validation.api.version>2.0.1.Final</validation.api.version>
+        <swagger.v3.version>2.0.8</swagger.v3.version>
+        <swagger.version>1.5.22</swagger.version>
+        <springfox.version>2.9.2</springfox.version>
+        <jackson.databind.nullable.version>0.1.0</jackson.databind.nullable.version>
+        <rest.assured.version>3.1.0</rest.assured.version>
+        <gson.version>2.6.1</gson.version>
+        <gson.fire.version>1.8.2</gson.fire.version>
+        <okio.version>1.13.0</okio.version>
+
+        <!-- Testing dependencies -->
+        <junit.version>4.12</junit.version>
+        <groovy.version>2.4.7</groovy.version>
+        <mockito-core.version>2.8.47</mockito-core.version>
+        <hamcrest-all.version>1.3</hamcrest-all.version>
+        <maven-plugin-testing-harness.version>3.3.0</maven-plugin-testing-harness.version>
+
+        <!-- Plugin dependency properties -->
+        <compiler.plugin.version>3.8.0</compiler.plugin.version>
+        <surefire.plugin.version>3.0.0-M3</surefire.plugin.version>
+        <faisafe.plugin.version>3.0.0-M3</faisafe.plugin.version>
+        <resources.plugin.version>3.1.0</resources.plugin.version>
+        <source.plugin.version>3.0.1</source.plugin.version>
+        <javadoc.plugin.version>3.1.0</javadoc.plugin.version>
+        <jar.plugin.version>3.1.1</jar.plugin.version>
+        <dependency.plugin.version>3.1.1</dependency.plugin.version>
+        <assembly.plugin.version>3.1.1</assembly.plugin.version>
+        <release.plugin.version>2.5.3</release.plugin.version>
+        <build.helper.plugin.version>3.0.0</build.helper.plugin.version>
+        <!--        <openapi.generator.plugin.version>3.3.4</openapi.generator.plugin.version>-->
+        <openapi.generator.plugin.version>4.0.0-beta</openapi.generator.plugin.version>
+
+        <!-- Gitflow Incremental Builder Properties -->
+        <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
+        <gib.baseBranch>HEAD</gib.baseBranch>
+        <gib.enabled>false</gib.enabled>
+        <gib.failOnError>false</gib.failOnError>
+    </properties>
+
+    <distributionManagement>
+        <!--
+        NOTE: The properties snapshots.repository.url and releases.repository.url should be defined in your .m2/settings.xml file.
+        -->
+<!--        <snapshotRepository>-->
```

Comment:
❓ Why are those commented out?

Reply:
I hadn't set those locally yet and wanted to get the maven build passing. I made a note to uncomment those.